experiment/objective.md

```diff
@@ -0,0 +1 @@
+Introduction to Mass Spectrometry Instrumentation and interpretation of Mass Spectra.
```

Comment:
Move the objective to the Aim file and remove objective.md since this is not used in teh experiment.

Reply:
Please remove this file.